### ISSUE TITLE
fftw_threads symbols

### ISSUE THREAD

**Issue by colinjcotter** [mpi4py/mpi4py-fft#13]
I get an import error when executing: from mpi4py_fft.fftw import fftw_xfftn

In [1]: from mpi4py_fft.fftw import fftw_xfftn
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-5178e9ea832e> in <module>
----> 1 from mpi4py_fft.fftw import fftw_xfftn

ImportError: dlopen(/Users/colincotter/firedrake-fftw2/lib/python3.8/site-packages/mpi4py_fft/fftw/fftw_xfftn.cpython-38-darwin.so, 2): Symbol not found: _fftw_cleanup_threads
  Referenced from: /Users/colincotter/firedrake-fftw2/lib/python3.8/site-packages/mpi4py_fft/fftw/fftw_xfftn.cpython-38-darwin.so
  Expected in: flat namespace
 in /Users/colincotter/firedrake-fftw2/lib/python3.8/site-packages/mpi4py_fft/fftw/fftw_xfftn.cpython-38-darwin.so

I have built fftw with --enable-threads --enable-shared (I'm asking PETSc to build it because I need to ensure that I'm using the right mpi library). libfftw3_threads can be found in the PETSc lib and I am adding the PETSc lib and include directories to FFTW_INCLUDE_DIR and FFTW_LIBRARY_DIR. How can I get mpi4py_fft to pick up these symbols?

**Comment by mikaem** [comment]
Hi Colin,
Not sure I can be of much help. Do you install mpi4py_fft from source? And you actually find the FFTW libraries during setup, but there are missing symbols? We do make use of both FFTW_INCLUDE_DIR and FFTW_LIBRARY_DIR in the setup.

**Comment by colinjcotter** [comment]
Yes, PETSc installs mpi4py_fft from source. It finds the FFTW libraries during setup, but trying to use mpi4py_fft then reports that stuff is missing.
________________________________

**Comment by colinjcotter** [comment]
OK, so I discovered that this issue only arises if I pip install mpi4py-fft from pypi and not if I clone the git repository and pip install from there.
